### ISSUE TITLE
Help with a Dialog

### ISSUE THREAD

**Issue by marioCoder94** [lumapps/lumX#430]
I have a controller and a HTML file with the same example in the page Lumx documentation
but not work for me, I get the dialog showing in the full screen

**Comment by marioCoder94** [comment]
<!DOCTYPE html>
<html ng-app="MonitoreoTablasApp">
  <head>
    <meta charset="utf-8">
    <title></title>
        <script src="bower_components/jquery/dist/jquery.min.js"></script>
        <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.9/angular.min.js"></script>

        <link rel="stylesheet" href="bower_components/lumx/dist/lumx.css">
        <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Roboto:300,400,500,700">
        <script src="controller.js"></script>
  </head>
  <body ng-controller="testController">

        testing

        <lx-button ng-click="Hola()">Open dialog</lx-button>

        <div class="p+">
    <lx-button ng-click="openDialog()">Open dialog</lx-button>
</div>

<lx-dialog id="test">
    <lx-dialog-header>
        <div class="toolbar bgc-primary pl++">
            <span class="toolbar__label tc-white fs-title">
                Lorem Ipsum
            </span>

            <div class="toolbar__right">
                <lx-button lx-size="l" lx-color="white" lx-type="icon" ng-click="vm.addPerson()">
                    <i class="mdi mdi-plus"></i>
                </lx-button>
            </div>
        </div>

        <lx-tabs lx-panes-id="lorem-ipsum"
                 lx-links="[{label: 'Lorem Ipsum 1'}, {label: 'Lorem Ipsum 2'}, {label: 'Lorem Ipsum 3'}]"
                 lx-layout="inline"
                 lx-theme="dark"
                 lx-color="primary"
                 lx-indicator="yellow"></lx-tabs>
    </lx-dialog-header>

    <lx-dialog-content>
        <lx-tabs-panes id="lorem-ipsum">
            <lx-tab-pane>
                <div class="p++">
                    <div ng-repeat="person in vm.people">
                        {{ person.name }} - {{ person.email }} - {{ person.age }}
                    </div>
                </div>
            </lx-tab-pane>

            <lx-tab-pane>
                <div class="p++">
                    <p>Lorem Ipsum Content 2</p>

                    <lx-text-field label="Description">
                        <textarea ng-model="textFields.description"></textarea>
                    </lx-text-field>
                </div>
            </lx-tab-pane>

            <lx-tab-pane>
                <p class="p++">Lorem Ipsum Content 3</p>
            </lx-tab-pane>
        </lx-tabs-panes>
    </lx-dialog-content>

    <lx-dialog-footer>
        <lx-button lx-color="black" lx-type="flat" lx-dialog-close>Cancel</lx-button>
        <lx-button lx-type="flat" lx-dialog-close>Ok</lx-button>
    </lx-dialog-footer>
</lx-dialog>


        <script src="bower_components/jquery/dist/jquery.js"></script>
        <script src="bower_components/velocity/velocity.js"></script>
        <script src="bower_components/moment/min/moment-with-locales.js"></script>
        <script src="bower_components/lumx/dist/lumx.js"></script>

  </body>
</html>

**Comment by marioCoder94** [comment]
var MonitoreoAPP = angular.module('MonitoreoTablasApp',[


    'lumx'
])

.controller('testController',function($scope,LxDialogService){

  $scope.Hola = function(){
    alert("wuy");
  }

  $scope.openDialog = function()
    {
        LxDialogService.open('test');
    }
})

**Comment by marioCoder94** [comment]
![sin titulo](https://cloud.githubusercontent.com/assets/14081810/14397916/8a492c32-fd9d-11e5-9e5e-3499dbde5da1.png)

**Comment by matmkian** [comment]
It's very difficult to read your markup. Can you fix it in your comment?
